src/app/components/SwapWidget/SwapStepOne.jsx

```diff
@@ -0,0 +1,162 @@
+import web3 from 'Services/Web3'
+import { container, header, section, body, submitButton, asset, icon, receive, swap } from './style.scss'
+import SwapIcon from 'Img/swap-icon.svg?inline'
+
+const SwapStepOne = ({ isPopUpOpen, handlePopUp, supportedAssets, 
+  deposit, receive, handleSelectedAsset, handleDepositSymbol, handleReceiveSymbol }) => (
+    <div className={classNames('container justify-content-center p-0', container)}>
+      <div className={header}>
+        <h1>Swap Instantly</h1>
+        <button onClick={() => handlePopUp('deposit')}>
+          <div className={asset}>
+            <CoinIcon key={deposit.symbol} symbol={deposit.symbol} style={{ width: 48, height: 48 }} className='m-1'/>
+            <h4>{deposit.symbol}</h4>
+            <p>Deposit</p>
+          </div>
+        </button>
+        <button className={swap} onClick={() => { handleDepositSymbol(receive.symbol); handleReceiveSymbol(deposit.symbol) }}><SwapIcon/></button>
+        <button onClick={() => handlePopUp('receive')}>
+          <div className={asset}>
+            <CoinIcon key={receive.symbol} symbol={receive.symbol} style={{ width: 48, height: 48 }} className='m-1'/>
+            <h4>{receive.symbol}</h4>
+            <p>Receive</p>
+          </div>
+        </button>
+      </div>
+      <div className={body}>
+        <SwapForm/>
+      </div>
+      <Modal size='lg' isOpen={isPopUpOpen} toggle={() => handlePopUp(null)} className='m-0 mx-md-auto' contentClassName='p-0'>
+        <ModalHeader toggle={() => handlePopUp(null)} tag='h4' className='text-primary'>
+          Add Asset
+        </ModalHeader>
+        <ModalBody>
+          {isPopUpOpen && (
+            <AssetSelector selectAsset={handleSelectedAsset} supportedAssetSymbols={supportedAssets}/>
+          )}
+        </ModalBody>
+      </Modal>
+    </div>
+)
+
+const validateAddress = (address) => {
+  if (!web3.utils.isAddress(address)) {
+    return <p>Please enter a receiving address</p>
+  }
+}
+
+const SwapForm = reduxForm({
+  form: 'swapWidget'
+})(({ handleSubmit, invalid }) => (
+  <Form onSubmit={handleSubmit}>
+    <div className={section}>
+      <ReduxFormField
+        row
+        className='gutter-3 align-items-center'
+        id='receive'
+        name='receive'
+        type='text'
+        placeholder='Receive Address'
+        autoFocus
+        labelProps={{ xs: '12', md: '4' }}
+        inputCol={{ xs:'12', md: true }}
+        validate={validateAddress}
+        inputClassName={receive}
+      />
+      <div style={{ position: 'relative' }}>
+        <ReduxFormField
+          row
+          className='gutter-3 align-items-center'
+          id='deposit'
+          name='deposit'
+          type='text'
+          placeholder='Deposit Amount (optional)'
+          autoFocus
+          labelProps={{ xs: '12', md: '4' }}
+          inputCol={{ xs:'12', md: true }}
+        />
+        <Expandable 
+          style={{ position: 'absolute', top: 16, right: 25 }} 
```

Comment:
I think this tooltip would look better to the right of the input rather than inside it